i18n/de.toml

```diff
@@ -145,7 +145,7 @@ selfConsumption = "Eigenverbrauch"
 avgPrice = "⌀ Preis"
 charged = "Geladen"
 co2 = "⌀ CO₂"
-duration = "Dauer"
+duration = "Ladezeit"
```

Comment:
Korrekt wäre Ladedauer. Es ist ja keine (Uhr)Zeit.

Reply:
Hilft es (im Session Log) hier den "Lade" Präfix zu ergänzen? Was für eine Dauer könnte es noch sein?

Reply:
Siehe Diskussion oben https://github.com/evcc-io/evcc/pull/9263#issuecomment-1666563645

> Was für eine Dauer könnte es noch sein?

Der Zeitraum vom Anstecke bis zum Abstecken. Also inkl. der Zeit in der nicht geladen wird.

Ich habs jetzt auf "Ladedauer" geändert. Im normalen Sprachgebrauch wird das sowieso Synonym verwendet und ich sehe hier keine Verwechselungsgefahr mit einem Zeitpunkt.